terraso_backend/apps/soil_id/graphql/soil_data.py

```diff
@@ -38,7 +64,50 @@ def slope_steepness_enum(cls):
         return cls._meta.fields["slope_steepness_select"].type()
 
 
+class ProjectSoilSettingsNode(DjangoObjectType):
+    class Meta:
+        model = ProjectSoilSettings
+        exclude = [
+            "deleted_at",
+            "deleted_by_cascade",
+            "id",
+            "created_at",
+            "updated_at",
+        ]
+
+    @classmethod
+    def measurement_units_enum(cls):
+        return cls._meta.fields["measurement_units"].type()
+
+    @classmethod
+    def depth_interval_preset_enum(cls):
+        return cls._meta.fields["depth_interval_preset"].type.of_type()
+
+
+class ProjectDepthIntervalNode(DjangoObjectType):
+    project = graphene.Field(ProjectNode, source="project__project", required=True)
+    depth_interval = graphene.Field(DepthInterval, required=True)
+
+    class Meta:
+        model = ProjectDepthInterval
+        exclude = [
+            "deleted_at",
+            "deleted_by_cascade",
+            "id",
+            "created_at",
+            "updated_at",
+            "depth_interval_start",
+            "depth_interval_end",
+        ]
```

Comment:
question: would it make sense to put the first five items in a constant and share between exclusion lists?

Reply:
yes for sure!